docs/client/python/pyspark.md

```diff
@@ -106,6 +106,32 @@ jdbcDF.write \
     .save()
 ```
 
+### Using as JDBC Datasource table with DDL
+
+From Spark 3.2.0, [`CREATE DATASOURCE TABLE`](https://spark.apache.org/docs/latest/sql-ref-syntax-ddl-create-table-datasource.html) is supported to create jdbc source with SQL.
+
+
+```python
+# create JDBC Datasource table with DDL
+spark.sql("""CREATE TABLE kyuubi_table USING JDBC
+OPTIONS (
+    driver='org.apache.hive.jdbc.HiveDriver',
+    url='jdbc:hive2://kyuubi_server_ip:port',
+    user='user',
+    password='password',
+    dbtable='testdb.some_table'
+)""")
+
+# read data to dataframe
+jdbcDF = spark.sql("SELECT * FROM kyuubi_table")
+
+# write data from dataframe
+df.writeTo("kyuubi_table")
```

Comment:
in scala, it should be `df.writeTo("kyuubi_table").append`, not sure python

Reply:
Thx for reminding me.  As `append` mode is not yet available since neither Hive nor Kyuubi's JDBC driver supports `addBatch`/`excuteBatch` mode, `overwrite` mode is added in this sample.